packages/editor/src/client/api/index.ts

```diff
@@ -1731,6 +1737,7 @@ export type User = {
   properties: Array<Properties>;
   roles: Array<UserRole>;
   subscription?: Maybe<UserSubscription>;
+  paymentProviderCustomers: Array<PaymentProviderCustomer>;
```

Comment:
What's this from?

Reply:
I think it comes from Nico's branch (the one he updated the payment type). Not sure why it comes in my code. @roosnic1 is it problematic ?

Reply:
@penk85 and @camilleory  Yeah i forgot to commit that file. Is fixed here https://github.com/wepublish/wepublish/pull/420 please review and once merged this should be gone.